content_test_filtering.py

```diff
@@ -55,11 +59,19 @@
         already_analysed.append(file_record["filepath"])
         # If change affected any other file -> analyse it
         changed_files.extend(diff_structure.affected_files)
-
     list_of_tests = connect_to_labels.get_labels(tests, options.output)
     if options.output == "json":
         logs.print_json(list_of_tests)
     else:
         logs.print_all_logs(list_of_tests, output_format=options.output_format)
-
+    # Save the updated controls to a file for syncing OSCAL catalog
+    logger.debug(f"The updated controls: {controls_files}")
+    if options.output == "json":
+        controls_updates = {"controls": controls_files}
+        try:
+            with open('controls_updates.json', 'w', encoding='utf-8') as file:
```

Comment:
Do you think that would be helpful to save the output for other changes, in profiles and maybe rules? If so, we could create a generic function to save files.

Reply:
Yeap. Sounds good. Updated. All the updates will be saved to output.json
`python content_test_filtering.py pr --base 4266a65aa118d6840c05d33d40a1612ad10bbf1c --remote_repo https://github.com/ComplianceAsCode/content --verbose --rule --profile --output  json 13176 > output.json`